src/qt/utilitydialog.cpp

```diff
@@ -57,9 +57,12 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
         ui->aboutMessage->setWordWrap(true);
         ui->helpMessage->setVisible(false);
     } else {
-        setWindowTitle(tr("Command-line options"));
-        QString header = "Usage: bitcoin-qt [command-line options] [URI]\n\n"
-                         "Optional URI is a Bitcoin address in BIP21 URI format.\n";
+        setWindowTitle(tr("options"));
+        QString header = "The bitcoin-qt application provides a graphical interface for interacting with " PACKAGE_NAME ".\n\n"
+                         "It combines the core functionalities of bitcoind with a user-friendly interface for wallet management, transaction history, and network statistics.\n\n"
+                         "It is suitable for desktop users preferring a graphical over a command-line interface.\n\n"
+                         "You can optionally specify a [BIP21URI] using the BIP21 URI format.\n\n"
```

Comment:
We should eliminate unnecessary specific BIP21 references in favour of just URI or maybe "payment URI" (see discussion of a new URI BIP on the horizon).